### PR TITLE
Adding Locke Data stuff (📯🌬)

### PR DESCRIPTION
**Pull request by stephlocke**
Adding my book and intro R course

### CODE REVIEW

File: content/learn.md [hunk 1]
```diff
@@ -44,6 +46,7 @@ DataCamp is an excellent way to improve your R skills, including the tidyverse.
 ## Workshops
 
   * [Mastering the Tidyverse](https://www.jumpingrivers.com/courses/22_r-tidyverse) by Jumping Rivers. A one day crash course covering tidyverse fundamentals. The course is a mixture of lectures, short exercises and longer tutorial questions. During the day, we'll cover dplyr, tidy data, tibbles, dates/times and string manipulation.
+  * [Introduction to R](https://itsalocke.com/courses/intro-to-r/) by Locke Data. A two day course covering data manipulation and reporting fundamentals using the tidyverse, rmarkdown and shiny. The course blends lectures, exercises, and practicals over two days to cover the 80% of work that almost everyone needs to do.
```
Comment (batpigandme) [inline content/learn.md]: 🤓 Tidyverse copy-editor here, can I just request an oxford comma in `rmarkdown, and shiny`? It's house style, and now that I see the one above is missing it, I'll change that, too!

File: content/learn.md [hunk 2]
```diff
@@ -24,6 +24,8 @@ We highly recommend pairing R4DS with the [RStudio cheatsheets](https://www.rstu
 * [Solutions and notes for R4DS](https://jrnold.github.io/r4ds-exercise-solutions/) 
   by Jeffrey B. Arnold. Work in progress.
 
+* [Data Manipulation in R](http://geni.us/datamanipulationir) by Steph Locke. Covers data manipalation in a tidyverse way.
```
Comment (hadley) [inline content/learn.md]: manipulation

File: content/learn.md (hunk 1 above)
Comment (stephlocke) [reply content/learn.md]: I didn't oxford comma because the prior one didn't. I'm *very happy* to add it in as it's the RIGHT WAY :)

**Comment by stephlocke** [comment]
Typo and grammar resolved 🙏

File: content/learn.md [hunk 3]
```diff
@@ -24,6 +24,8 @@ We highly recommend pairing R4DS with the [RStudio cheatsheets](https://www.rstu
 * [Solutions and notes for R4DS](https://jrnold.github.io/r4ds-exercise-solutions/) 
   by Jeffrey B. Arnold. Work in progress.
 
+* [Data Manipulation in R](http://geni.us/datamanipulationir) by Steph Locke. Covers data manipulation in a tidyverse way.
```
Comment (hadley) [inline content/learn.md]: Should that be `datamanipulationinr`?

Comment (stephlocke) [reply content/learn.md]: Made the link work :) http://geni.us/datamanipulationir

**Comment by hadley** [comment]
Thanks!

**Pull request merged**